swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, fabric="Unknown",condition=0,age=0):
+        super().__init__(id,condition)
+        self.fabric = fabric
+        self.age = age
```

Comment:
If all the children classes will have an attribute called `age` could we move this attribute into the parent class `Item`?